extras/paid-courses.md

```diff
@@ -86,6 +86,10 @@
 ### Artificial Intelligence
 
 ### Machine Learning
+Name | Provider
+:-- | :--:
+[Machine Learning by Andrew Ng](https://www.coursera.org/learn/machine-learning) | [Coursera](https://www.coursera.org)
+[Machine Learning Engineer Nanodegree](https://www.udacity.com/course/machine-learning-engineer-nanodegree--nd009) | [Udacity](https://www.udacity.com)
```

Comment:
Ok i will wait for @ericdouglas to review this and then make necessary changes. Thanks @benbrandt  for review and pointing the mistakes :+1: